docker/Dockerfile.cuda9.deps

```diff
@@ -0,0 +1,19 @@
+FROM alpine
```

Comment:
I just want to point out, that the base distribution changes here, from CentOS to alpine. We're sure, that it's OK?

Reply:
Yes. This image is just a bag to keep cuda inside. We don't want to rebuild it when manylinux changes.